setup.py

```diff
@@ -34,7 +34,7 @@
         "resolvelib>=0.8.0",
         "html5lib>=1.1",
         "CacheControl[filecache]>=0.12.10",
-        "cyclonedx-python-lib>=1.0.0,<2.0.0",
+        "cyclonedx-python-lib>=1.0.0,<3.0.0",
```

Comment:
While we're at it, should we bump the lower requirement to `2.0.0` so that we're only supporting one major version at a time?

Reply:
I think we should try to support as broad of a range as possible without requiring significant effort.

Reply:
Yep, agreed. We can constrain if and when changes between major versions require us to e.g. import different classes between versions.